Voga/Presentation/Login/View/TextField.swift

```diff
@@ -0,0 +1,76 @@
+//
+//  File.swift
+//  Voga
+//
+//  Created by Ariadne Bigheti on 12/11/21.
+//
+
+import Foundation
+import UIKit
+
+class TextField : UIView, UITextFieldDelegate {
+    
+    init() {
+        super.init(frame: .zero)
+        buildView()
+    }
+
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    private lazy var title: VogaLabel = {
+        let vogaLabel = VogaLabel(textStyle: .Body)
+        vogaLabel.textColor = .appColor(.black)
+
+        return vogaLabel
+    }()
+
+    private lazy var textField: UITextField = {
+        let field = UITextField()
+        field.allowsEditingTextAttributes = true
+        field.font = UIFont(name: "AtkinsonHyperlegible-Regular", size: 15)
+        field.placeholder = "field"
+        field.backgroundColor = .appColor(.background)
+        field.borderStyle = UITextField.BorderStyle.roundedRect
+        field.autocorrectionType = UITextAutocorrectionType.no
+        field.keyboardType = UIKeyboardType.default
+        field.isSecureTextEntry = false
+        field.returnKeyType = UIReturnKeyType.done
+        field.clearButtonMode = UITextField.ViewMode.whileEditing
+        field.contentVerticalAlignment = UIControl.ContentVerticalAlignment.center
+        field.delegate = self
+        
+        return field
+    }()
+
+    func configure(with title: String, with field: String) {
```

Comment:
This really isn't a good signature for a method, using `with` in both parameters is really confusing for someone seeing the code. You should use these kinds of naming for methods when the parameter name and the parameter value would be redundant, like `logInUser(with user: User)`, because when someone uses this method it would be called like: `logInUser(with: user)`. This is not the case here, so you could name this 
```suggestion
    func configure(title: String, placeholder: String) {
```

Reply:
Also, since most of the times I saw you guys use this TextField you used the same name both for the title and the placeholder, like `configure(with: "name", with: "name")` maybe you could also make the placeholder optional and in case it is nil you use the same value both for title and the placeholder.

Reply:
Also, a better practice here would be to use a model to configure this, it would be something like:
```suggestion
    struc TextFieldModel {
    var title: String
    var placeholder: String?
    }
    
    ...
    configure(with model: TextFieldModel) {
    self.title.vogaText = model.title
    guard let placeholder = model.placeholder else {
    textField.placeholder = model.title
    }
        textField.placeholder = placeholder
    }
```